eclass/linux-mod-r1.eclass

```diff
@@ -904,6 +948,21 @@ _modules_process_depmod.d() {
 	)
 }
 
+# @FUNCTION: _modules_process_dracut.conf.d
+# @USAGE: <module>...
+# @INTERNAL
+# @DESCRIPTION:
+# Create dracut.conf.d snippet defining if module should be included in the
+# initramfs.
+_modules_process_dracut.conf.d() {
+	(
+		insinto /usr/lib/dracut/dracut.conf.d
+		[[ ${MODULES_INITRAMFS_IUSE} ]] && use ${MODULES_INITRAMFS_IUSE#+} &&
+			: add || : omit
+		newins - 10-${PN}.conf <<<"${_}_drivers+=\" ${*%.ko} \""
```

Comment:
Oh, one question.. what kind of files are dracut.conf? Custom configs, /bin/sh, or /bin/bash?

Asking because posix sh doesn't support `+=`

Reply:
It's all bash: https://github.com/dracut-ng/dracut-ng/blob/c5cbdaf3050f46c446fe736032ae872376f6d365/dracut.sh#L942

The dracut.conf manual also references `+=` for config settings that list modules/drivers/firmware etc.